seed/challenges/02-data-visualization-certification/react.json

```diff
@@ -1,30 +1,19 @@
 {
   "name": "React",
   "order": 14,
-  "isComingSoon": true,
+  "isComingSoon": "true",
```

Comment:
`isComingSoon` is defined as `boolean`, https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/common/models/challenge.json#L36-L38. Not sure if this makes sense but I think it's better remove double quotes. The same thing everywhere.